tests/components/stream/test_worker.py

```diff
@@ -247,8 +249,12 @@ async def test_stream_worker_success(hass):
 async def test_skip_out_of_order_packet(hass):
     """Skip a single out of order packet."""
     packets = list(PacketSequence(TEST_SEQUENCE_LENGTH))
+    # for this test, make sure the out of order index doesn't happen on a keyframe
+    out_of_order_index = OUT_OF_ORDER_PACKET_INDEX
+    if packets[out_of_order_index].is_keyframe:
+        out_of_order_index += 1
     # This packet is out of order
-    packets[OUT_OF_ORDER_PACKET_INDEX].dts = -9090
+    packets[out_of_order_index].dts = -9090
```

Comment:
Maybe just `assert not packets[out_of_order_index].is_keyframe` here just for clarity though I realize it should always be true.

Reply:
Sure